src/webnwb_benchmark/benchmark.py

```diff
@@ -34,6 +31,7 @@ def do_GET(self):
 class WebNWBBenchmarks:
     
     def __init__(self, html_path: Path):
+        """Initialize the instance properties"""
```

Comment:
Can you add an input description (numpy format) for `html_path` with some context of how to get or structure it?